optimum/intel/openvino/modeling_decoder.py

```diff
@@ -630,22 +654,12 @@ def _from_pretrained(
 class OVBloomForCausalLM(OVModelForCausalLM):
     # Adapted from transformers.models.bloom.modeling_bloom.BloomForCausalLM.prepare_inputs_for_generation
     def prepare_inputs_for_generation(self, input_ids, past_key_values=None, **kwargs):
-        attention_mask = kwargs.get("attention_mask", None)
-        use_cache = kwargs.get("use_cache", None)
-
         # only last token for input_ids if past is not None
         if past_key_values and not self.stateful:
             # the cache may be in the stardard format (e.g. in contrastive search), convert to bloom's format if needed
             if past_key_values[0][0].shape[0] == input_ids.shape[0]:
                 past_key_values = self._convert_to_bloom_cache(past_key_values)
-
```

Comment:
@eaidova could you also add a test for verify that calling `.generate()` twice in a row gives expected results ? (`self._past_length` correctly re-initialized)

Reply:
also current tests are failing due to what seems to be a mismatch in the shape of generated `position_ids`